libs/wire-api/src/Wire/API/Routes/Public/Galley/Messaging.hs

```diff
@@ -98,9 +99,31 @@ type MessagingAPI =
                     (PostOtrResponses MessageSendingStatus)
                     (Either (MessageNotSent MessageSendingStatus) MessageSendingStatus)
            )
+    :<|> Named
+           "post-proteus-broadcast@v5"
+           ( Summary "Post an encrypted message to all team members and all contacts (accepts only Protobuf)"
+               :> From 'V5
+               :> Description PostOtrDescription
+               :> ZLocalUser
+               :> ZConn
+               :> CanThrow 'TeamNotFound
+               :> CanThrow 'BroadcastLimitExceeded
+               :> CanThrow 'NonBindingTeam
```

Comment:
Perhaps I'm blind but it looks like you forgot to actually add the additionally error label?

Reply:
you are not blind, but correct!  thanks-